### ISSUE TITLE
Version number

### ISSUE THREAD

**Issue by Agusanz** [znc/znc#819]
On the version "znc-1.6.0-alpha1" it displays instead "znc-1.5.0-alpha1".
You notice this when executing: znc or znc -v.

**Comment by DarthGandalf** [comment]
http://wiki.znc.in/User:Psychon/HowToRelease says to update the version number only for RC...

**Comment by Mikaela** [comment]
I am just seeing ZNC - 1.5.0-git-485-230a39f - http://znc.in, was it changed in some newer commit or is that alpha somewhere outside of git?

**Comment by DarthGandalf** [comment]
It's in topic of #znc, and on website http://znc.in/

**Comment by Mikaela** [comment]
Expired issue?

**Comment by jpnurmi** [comment]
I agree that "znc-1.5.0-alpha1" is misleading. I was the alpha release of 1.6.0.
